TranslucentTB/main.cpp

```diff
@@ -606,13 +622,9 @@ bool singleProc()
 int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPreInst, LPSTR pCmdLine, int nCmdShow)
 {
 	if (singleProc()) {
-
-		// TODO Usually the command line overrides the config file, so these should be reversed.
-		// Still, we check the --config parameter on the command line, so that should be handled
-		// before the config file. Needs two passes on the command line arguments, lets leave that
-		// for later.
-		ParseCmdOptions(); //command line argument settings
-		ParseConfigFile(L"config.cfg"); //config file settings
+		ParseCmdOptions(true); // Command line argument settings, config file only
+		ParseConfigFile(L"config.cfg"); // Config file settings
```

Comment:
It still loads config.cfg unconditionally. You should set a flag inside ParseCmdOptions that indicates whether any --config option was passed on the command line. Only if this flag is not set should you load config.cfg from WinMain.